traffic_portal/app/src/common/modules/table/types/table.types.tpl.html

```diff
@@ -34,18 +47,19 @@
             <thead>
                 <tr class="headings">
                     <th>Name</th>
-                    <th>Use In Table</th>
                     <th>Description</th>
+                    <th>Last Updated</th>
+                    <th>Use In Table</th>
```

Comment:
I hate this name, because it's tying our UI directly to object representation i nthe database, which is meant to be opaque to Traffic Portal. But I don't have a better suggestion, so if you don't either, I guess it's fine.

Reply:
yes, i'm not a fan either. it's a bad practice imo that bubbled up to the UI